code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -232,3 +232,7 @@
 	var/fire_luminosity = 0 //Luminosity of the current fire while burning
 
 	var/butchery_progress = 0
+
+	//Trap vars
+	var/trapchoices = list()
+	var/gastier = 5
```

Comment:
should the default really be a 5 here? wouldn't 1 be more appropriate so people don't get confused when adding a new caste?

Reply:
Ah, will fix it, sorry!